src/Shared/Components/GraphVisualizer/constants.ts

```diff
@@ -0,0 +1,31 @@
+import { NodeTypes, Viewport } from '@xyflow/react'
+
+import { DropdownNode, IconNode, TextNode } from './components'
+
+const nodeTypes = {
+    iconNode: IconNode,
+    textNode: TextNode,
+    dropdownNode: DropdownNode,
+}
```

Comment:
Can be const for all
```suggestion
} as const
```

Reply:
not needed